files/ubuntu-20.04/sge_preinstall.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
```

Comment:
This file looks identical to the Ubuntu18 version. Can we use the same file under `files/ubuntu/` for both OSs? Same comment applies for other files that might be the same for Ubuntu OSs

Reply:
Done